addons/message_plan_upgrade_14d/conditions.js

```diff
@@ -0,0 +1,40 @@
+(function(api, condition) {  
+  //Show this message after 14 days, but not after 87 days (there is a separate message for that)
+
+  //There are 2 identical (content-wise), yet separate messages because we want to give the illusion that we are 
+  //re-showing the same message 87 days in (by hiding the 14 day one and showing the 87 day one) without having to
+  //un-read or un-dismiss a message
```

Comment:
Why would un-reading / un-dismissing be bad here?

Reply:
I remember investigating this route, but I don't remember why it wasn't working. This was before we had the discussion about timers. Let me reinvestigate